framework/decode/stat_decoder_base.h

```diff
@@ -84,32 +86,34 @@ class StatDecoderBase : public ApiDecoder
                                              format::HandleId surface_id,
                                              uint32_t         width,
-                                             uint32_t         height)
+                                             uint32_t         height) override
     {}
 
     virtual void DispatchResizeWindowCommand2(format::ThreadId thread_id,
                                               format::HandleId surface_id,
                                               uint32_t         width,
                                               uint32_t         height,
-                                              uint32_t         pre_transform)
+                                              uint32_t         pre_transform) override
     {}
 
-    virtual void DispatchCreateHardwareBufferCommand(format::ThreadId                                    thread_id,
-                                                     format::HandleId                                    memory_id,
-                                                     uint64_t                                            buffer_id,
-                                                     uint32_t                                            format,
-                                                     uint32_t                                            width,
-                                                     uint32_t                                            height,
-                                                     uint32_t                                            stride,
-                                                     uint64_t                                            usage,
-                                                     uint32_t                                            layers,
-                                                     const std::vector<format::HardwareBufferPlaneInfo>& plane_info)
+    virtual void
+    DispatchCreateHardwareBufferCommand(format::ThreadId                                    thread_id,
+                                        format::HandleId                                    memory_id,
+                                        uint64_t                                            buffer_id,
+                                        uint32_t                                            format,
+                                        uint32_t                                            width,
+                                        uint32_t                                            height,
+                                        uint32_t                                            stride,
+                                        uint64_t                                            usage,
+                                        uint32_t                                            layers,
+                                        const std::vector<format::HardwareBufferPlaneInfo>& plane_info) override
     {}
```

Comment:
Talking of clang-format versions elsewhere, this change in formatting seems a bit random 🤷 . Just sayin.

Reply:
Yup.